### DISCUSSION TITLE
Using markdonw in text of a checkbox don't work

### DISCUSSION THREAD

**Discussion by Yivan** [requarks/wiki#3710] [Error / Bug Report]
**Describe the bug**
Using markdonw in text of a checkbox don't work, in preview mode and on the final page too.
Bold, etc. don't work too.

**To Reproduce**
On an edit page with markdown editor, fill:

```
* [ ] The sun is high in the `sky` + `here`
* [ ] The sun is high in the sky + here
```

Result is:

![image](https://user-images.githubusercontent.com/9025839/72288309-102b3100-3649-11ea-8f07-908e9d9c23b1.png)

Same bug on final page rendering, than preview mode.

**Expected behavior**
A correct markdown (we used GitLab wiki before, and this worked correctly).
Here the checkbox is replaced by a list and the "+" double the sentence strangly.

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2
 - Database engine: postgres

Thanks.

**Comment by isiah-lloyd** [comment]
I could reproduce this using the latest development version. I tested your example using the markdown parser we use's [demo page](https://markdown-it.github.io/), so the issue seems to be with our implementation.
